### DISCUSSION TITLE
Support Next js AppRouter ?

### DISCUSSION THREAD

**Discussion by friday-sds** [gladly-team/next-firebase-auth#653] [Q&A]
Is This Library Supporting Next Js AppRouter  Architecture . ?

**Comment by kmjennison** [comment]
Not yet:
https://github.com/gladly-team/next-firebase-auth/issues/568

**Reply by lazycipher** [reply]
@kmjennison, after the v1 beta release, do we have the app router support now?

**Reply by kmjennison** [reply]
No. Please follow that issue for updates on it.
